subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -485,8 +485,10 @@ uint8_t ll_cig_parameters_commit(uint8_t cig_id, uint16_t *handles)
 
 		if (!cig->central.test) {
 			/* Make sure specified Max_Transport_Latency is not exceeded */
-			LL_ASSERT(c_latency <= cig->c_latency);
-			LL_ASSERT(p_latency <= cig->p_latency);
+			if (c_latency > cig->c_latency || p_latency > cig->p_latency) {
+				/* Invalid Max_Transport_Latency */
+				return BT_HCI_ERR_INVALID_PARAM;
```

Comment:
After CIG/CIS creation, it is not allowed to return without cleaning up. This fix needs the cleanup code of #60205

Reply:
And #60205 has the same fix.